test/unit/argument_parser/detail/CMakeLists.txt

```
@@ -1,4 +1,7 @@
-add_definitions(-DSEQAN_INCLUDE_DIR="${SEQAN3_INCLUDE_DIR}")
+# seqan3_test_unit_BINARY_DIR is needed for debian builds
+find_path (SEQAN3_TEST_LICENSE_DIR NAMES LICENSE.md HINTS "${SEQAN3_CLONE_DIR}" "${seqan3_test_unit_BINARY_DIR}/..")
```

I want to remove hard-coded paths. 

```suggestion
find_path (SEQAN3_TEST_LICENSE_DIR NAMES LICENSE.md HINTS "${SEQAN3_CLONE_DIR}")
```

You can either add `cmake -DCMAKE_INCLUDE_PATH=..` to influence `find_path` to find `LICENSE.md`, or you can just copy then `LICENSE.md` into the ` test/unit/argument_parser/detail/` directory as a pre-processing step.

See https://cmake.org/cmake/help/v3.16/variable/CMAKE_INCLUDE_PATH.html#variable:CMAKE_INCLUDE_PATH